### DISCUSSION TITLE
Hardhat zksync node(21 alpha compiler doesn't work)

### DISCUSSION THREAD

**Discussion by novaknole** [zkSync-Community-Hub/zksync-developers#535] [Compilers]
### Which compiler are you using?

- [X] zksolc
- [ ] zkvyper

### Which version of Solidity or Vyper?

0.8.17

### zksolc/zkvyper version?

latest

### Explain the issue

```bash
It seems like zksync team pushed new compiler version into zksync hardhat node. 

5 days ago, it was working fine, but when I ran the node yesterday, it downloaded alpha-21 version (in the past, it was using 19th version). and in alpha-21 version, contracts can't be deployed because it says: block gas limit too much.
```

**Comment by MexicanAce** [comment]
Hi @novaknole , thank you for the report! The team is taking a look at the latest updates with `v0.1.0-alpha.21` and investigating similar gas issues. Will post here when a new release has resolved this issue. For now, it's possible to reference version 19 with `npx hardhat node-zksync --tag 0.1.0-alpha.19`

**Reply by novaknole** [reply]
Thank you. Appreciate the quick response.

**Reply by MexicanAce** [reply]
We've just release v22 that should have some fixes, can you try again with this updated version please?

https://github.com/matter-labs/era-test-node/releases/tag/v0.1.0-alpha.22
